markdown/blog/asyncapi-mascot-redesign.md

```diff
@@ -0,0 +1,74 @@
+---
+title: AsyncAPI Mascot Redesign: The Biography of Eve and Chan
+date: 2025-02-25T06:00:00+01:00
+type: Community
+tags:
+  - Mascot
+  - Design
+  - Marketing
+  - Community
+cover: /img/posts/mascot-banner.webp
+authors:
+  - name: Victoria Nduka
+    photo: /img/avatars/nwanduka.webp
+    link: https://www.linkedin.com/in/victorianduka/
+    byline: User Experience Designer
+excerpt: 'The story behind Eve and Chan—how they started, why they changed, and where they’re headed next.'
+---
+
+# AsyncAPI Mascot Redesign: The Biography of Eve and Chan
+
+- [The Birth of Eve and Chan](#the-birth-of-eve-and-chan)
+- [Why the Redesign?](#why-the-redesign)
+- [Meet Eve and Chan](#meet-eve-and-chan)
+- [What’s Next for Eve and Chan?](#whats-next-for-eve-and-chan)
+
+AsyncAPI’s mascots, Eve and Chan, have had a brand refresh, and I’m proud to say that I had a hand in their redesign. Now, seeing the community use them in different ways just warms my heart. Any designer will tell you that nothing beats the feeling of seeing your work come to life beyond the Figma canvas, and that’s exactly how I feel about the new mascots.
+
+Throughout the redesign process, I kept asking myself questions to ensure the new look stayed true to their original purpose. Whose idea was it to have the mascots? Why the names Eve and Chan? Who designed them originally? Who are Eve and Chan, really?
+
+As it turns out, I’m not the only one with these questions. Within and outside the AsyncAPI community, folks have often wondered about the origins of these characters. So, I recently sat down with Fran Mendez, the founder of AsyncAPI, to get their backstory. This is the story of Eve and Chan—their biography, if you will.
+
+
+## The Birth of Eve and Chan
+
+It was Fran’s idea to have a mascot that people could easily associate with the AsyncAPI brand. Most open-source projects and companies use mascots to reinforce their brand identity, so why not AsyncAPI? You know how Linux has Tux the penguin, and GitHub has Octocat? Fran wanted something like that for AsyncAPI.
```

Comment:
```suggestion
[Fran Méndez’s](https://www.linkedin.com/in/fmvilas) idea was to have a mascot that people could easily associate with the AsyncAPI brand. Most open-source projects and companies use mascots to reinforce their brand identity, so why not AsyncAPI? Since Linux has Tux, the Penguin, and GitHub has Octocat, Fran wanted something like that for AsyncAPI.
```